src/NuGet.Jobs.GitHubIndexer/GitRepoSearchers/GitHub/GitHubSearchApiResponse.cs

```diff
@@ -0,0 +1,22 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System;
+using Octokit;
+
+namespace NuGet.Jobs.GitHubIndexer
+{
+    public class GitHubSearchApiResponse
+    {
+        public GitHubSearchApiResponse(SearchRepositoryResult result, DateTimeOffset date, DateTimeOffset throttleResetTime)
+        {
+            Result = result;
```

Comment:
Don't forget a null check here:

```suggestion
            Result = result ?? throw new ArgumentNullException(nameof(result));
```